src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Created by raynoldng on 5/2/18.
+ */
+public class Block extends AddressProperty {
+    public Block (String value) {
+        super(value);
```

Comment:
You can consider moving the validation and equality check to the new classes you have created.